sdk/aqueduct/client.py

```diff
@@ -188,19 +196,39 @@ def create_param(self, name: str, default: Any, description: str = "") -> BaseAr
                 The name to assign this parameter.
             default:
                 The default value to give this parameter, if no value is provided.
-                Every parameter must have a default.
+                Every parameter must have a default. If decided to use local data,
+                a path to the local data file must be specified.
             description:
                 A description of what this parameter represents.
-
+            use_local:
+                whether this parameter uses local data source or not.
+            as_type:
+                The expected type of the local data. Currently Local Data has support for ArtifactType.TABLE
```

Comment:
Can we replace the second sentence with "Only supported types are ArtifactType.TABLE and ArtifactType.IMAGE